Sources/ArgumentParser/Parsable Properties/Argument.swift

```diff
@@ -204,44 +204,53 @@ extension Argument {
   
   /// Creates a property that reads an array from zero or more arguments.
   ///
-  /// The property has an empty array as its default value.
+  /// The property has an empty array as its default value if the `initial`
+  /// parameter is not specified.
   ///
   /// - Parameters:
+  ///   - initial: A default value to use for this property. If `initial` is
+  ///     `nil`, this option defaults to an empty array.
   ///   - parsingStrategy: The behavior to use when parsing multiple values
   ///     from the command-line arguments.
   ///   - help: Information about how to use this argument.
   public init<Element>(
+    default initial: Value? = nil,
```

Comment:
For all these we should be able to just use `[]` as the default instead of using the optional. Since that’ll be visible in the declaration, we don’t need to cover it in the documentation, either.

Reply:
Of course. I was trying to match the other initializers, and thought about this right after the PR.